pkgs/development/embedded/blisp/default.nix

```diff
@@ -31,12 +32,16 @@ stdenv.mkDerivation {
     "-DBLISP_USE_SYSTEM_LIBRARIES=ON"
   ];
 
+  passthru.tests.version = testers.testVersion {
+    package = finalAttrs.finalPackage;
+    command = "blisp --version";
+    version = "v${finalAttrs.version}";
+  };
+
   meta = with lib; {
-    description = "ISP tool & library for Bouffalo Labs RISC-V Microcontrollers and SoCs";
+    description = "An In-System-Programming (ISP) tool & library for Bouffalo Labs RISC-V Microcontrollers and SoCs";
     license = licenses.mit;
     homepage = "https://github.com/pine64/blisp";
     maintainers = [ maintainers.fortuneteller2k ];
```

Comment:
Can you add `mainProgram = "blisp";` ?

Reply:
I think we cat get away without setting `mainProgram` here given the only executable the package emits is equal to the `pname`. So `nix run` will do the right thing and `testVersion` as commented above will also fall back to using `pname`.

Reply:
That policy has recently changed #246386 .